### ISSUE TITLE
Gem release

### ISSUE THREAD

**Issue by manuelmeurer** [benignware/schedulable#38]
Could you release a new gem version please?
Seems like there are a lot of fixes and new features since the last release in November 2015.

**Comment by bjgaynor** [comment]
Agreed, there's a lot of good stuff that seems to be just waiting for a release.

**Comment by yshmarov** [comment]
Please do!
